packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -2339,7 +2339,14 @@ export const audiusBackend = ({
         })
         .flat()
         .filter(removeNullable)
-      const userId = decodeHashId(notification.actions[0].specifier) as number
+      // playlist owner ids are the specifier of the playlist create notif
+      const userId =
+        entityType !== Entity.Track
```

Comment:
can entity type be something besides track & collection? will this break if so?

Reply:
'create' types only happen on track and playlist handler

maybe use === and reverse the logic for clarity

Reply:
also `entityType` var is created on line 2327 and only set in 3 places, once on initialization, and in lines 2332 and 2337 where its all either set to track, playlist, or album

Reply:
the code could definitely be more safeguarded